ports/libsimpleservo/capi/src/lib.rs

```diff
@@ -239,7 +297,11 @@ unsafe fn init(
     wakeup: extern "C" fn(),
     callbacks: CHostCallbacks,
 ) {
-    init_logger();
+    if !opts.vslogger_mod_list.is_null() && opts.vslogger_mod_size > 0 {
+        opts.build_mod_list_vec();
+    }
```

Comment:
Let's do the following instead (using the [std::slice api](https://doc.servo.org/std/slice/fn.from_raw_parts.html)):
```
let logger_modules = if opts.vslogger_mod_list.is_null() {
    &[]
} else {
    slice::from_raw_parts(opts.vslogger.mod_list, opts.vslogger_mod_size)
};
```
Then we can make init_logger take a `&[*const c_char]` argument, which will be nicer to work with.

Reply:
Implemented.